### ISSUE TITLE
Get tests passing on CMS-Test under PHP 8.1

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#11800]
## Description

See failures [here](http://jenkins.vfs.va.gov/job/testing/job/cms-test-post-deploy-tests-staging/).

## Acceptance Criteria
- [x] All test pass as expected under PHP 8.1:
  - [x] `va/tests/content-build-gql`
  - [x] `va/tests/behat`
  - [x] `va/tests/phpunit`
  - [x] `va/tests/phplint`
  - [x] `va/tests/cypress`

**Comment by ndouglas** [comment]
## `content-build-gql`

```
Finished 728 queries in 262s with 19740 pages
https://test.staging.cms.va.gov response time: : 4:22.914 (m:ss.mmm)
Total time to load content from GraphQL: 4:26.885 (m:ss.mmm)
Drupal successfully loaded!
Successfully piped Drupal content into Metalsmith!
Step 2 end [326371ms]: Get Drupal content
                                            Step summary                                             
  Step         Description         Time Elapsed   Heap Change   Total Heap   RSS Change   Total RSS  
                                                                                                     
     0   Build static data files   2512ms         0.63mB        34.12mB      1.52mB       85.36mB    
     1   Get Drupal content        326371ms       1746.01mB     1780.14mB    2163.97mB    2249.32mB  

Peak RSS used: 0mB

GraphQL queries have been run
Done in 331.40s.
```

## `behat`

```
80 scenarios (80 passed)
589 steps (589 passed)
6m58.26s (125.46Mb)

real    6m58.791s
user    0m28.910s
sys     0m1.925s
```

## `phpunit`

```
Testing tests/phpunit
PHP Deprecated:  Return type of GuzzleHttp\Cookie\CookieJar::count() should either be compatible with Countable::count(): int, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /var/www/cms/docroot/vendor/guzzlehttp/guzzle/src/Cookie/CookieJar.php on line 220
PHP Deprecated:  Return type of GuzzleHttp\Cookie\CookieJar::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /var/www/cms/docroot/vendor/guzzlehttp/guzzle/src/Cookie/CookieJar.php on line 225
.........SSS...................................................  63 / 634 (  9%)
............................................................... 126 / 634 ( 19%)
............................................................... 189 / 634 ( 29%)
............................................................... 252 / 634 ( 39%)
............................................................... 315 / 634 ( 49%)
............................................................... 378 / 634 ( 59%)
............................................................... 441 / 634 ( 69%)
............................................................... 504 / 634 ( 79%)
...........................S................................... 567 / 634 ( 89%)
............................................................... 630 / 634 ( 99%)
....                                                            634 / 634 (100%)

Time: 9.39 minutes, Memory: 3.12 GB

OK, but incomplete, skipped, or risky tests!
Tests: 634, Assertions: 1306, Skipped: 4.

real    9m26.674s
user    6m53.613s
sys     0m9.990s
```

**Comment by ndouglas** [comment]
![Screen Shot 2022-12-06 at 8 56 32 AM](https://user-images.githubusercontent.com/1318579/205931257-cbcce14d-824a-4177-bc6d-ad5c75a7c1a2.png)

That's in here:

![Screen Shot 2022-12-06 at 8 57 03 AM](https://user-images.githubusercontent.com/1318579/205931436-0591e434-fd05-4dda-90ac-70983dc9bb21.png)

The element just does not get populated:

![Screen Shot 2022-12-06 at 8 57 46 AM](https://user-images.githubusercontent.com/1318579/205931572-d1665320-c846-45ed-ad52-6dc40cbcf301.png)

**Comment by ndouglas** [comment]
With Swirt's fix for #11484, tests are passing on the PR branch (and a branch off that PR) that I've tested.
